pkg/config/protocol.go

```diff
@@ -61,6 +63,8 @@ const (
 // ParseProtocol from string ignoring case
 func ParseProtocol(s string) Protocol {
 	switch strings.ToLower(s) {
+	case "auto", "":
+		return ProtocolAuto
```

Comment:
You don’t need this. Do you ? Or atleast where are you assigning the auto protocol at all ?

Reply:
The idea is to automatically take ports whose protocols are unknown and classify them as auto.

Reply:
I think we might need to allow users to configure the protocol as `auto` explicitly. 

`auto` ports will contain both ports whose protocol is unknown and those who are configured with `auto` explicitly.

Reply:
Can we not add a new user-facing protocol type for now? We can always add it in later if we decide we need it. For now, let's just do this for unknown protocols.

Reply:
Removed user-facing protocol.